templates/back-office/packing-slip-content.hypr

```diff
@@ -551,9 +551,9 @@
                 <tr>
                     <td>
                         {% for item in model.items %}
-                        {% if substituteItem.originalLineId == item.lineId %}
-                        <div class="product-name">{{ item.name }}</div>
-                        {% endif %}
+                            {% if substituteItem.originalLineId == item.lineId %}
```

Comment:
Are we sure about comparing 'originalLineId ' with 'lineId'?
As we refer to the 'shipmentId' field from the order's model which is equivalent to 'lineId' from the shipment model.